src/lib/core/CHIPConfig.h

```diff
@@ -1368,3 +1364,80 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #ifndef CHIP_CONFIG_MAX_SUBSCRIPTION_RESUMPTION_STORAGE_CONCURRENT_ITERATORS
 #define CHIP_CONFIG_MAX_SUBSCRIPTION_RESUMPTION_STORAGE_CONCURRENT_ITERATORS 2
 #endif
+/**
+ * @brief Indicates the absence of a Scene table entry.
+ */
+#ifndef CHIP_CONFIG_SCENES_TABLE_NULL_INDEX
+#define CHIP_CONFIG_SCENES_TABLE_NULL_INDEX 0xFF
+#endif
+
+/**
+ * @brief Value used when setting or getting the endpoint in a Scene table
+ * entry.  It indicates that the entry is not in use.
+ */
+#ifndef CHIP_CONFIG_SCENES_TABLE_UNUSED_ENDPOINT_ID
+#define CHIP_CONFIG_SCENES_TABLE_UNUSED_ENDPOINT_ID 0x00
+#endif
+/**
+ * @brief The maximum number of scenes according to spec
+ */
+#ifndef CHIP_CONFIG_SCENES_MAX_NUMBER
+#define CHIP_CONFIG_SCENES_MAX_NUMBER 16
+#endif
+
+/**
+ * @brief Maximum length of Scene names, not including the length byte.
```

Comment:
Why is length byte mentioned? Isn't that an implementation detail of the storage?

Reply:
Removed in latest commit.